tests/Admin/AdminTest.php

```diff
@@ -2573,6 +2630,7 @@ public function testGetDataSourceIterator(): void
                 return $label;
             });
 
+        $this->expectDeprecation('Using "Sonata\AdminBundle\Admin\AbstractAdmin::getDataSourceIterator" without setting a "Sonata\AdminBundle\Exporter\DataSourceInterface" is deprecated since sonata-admin/admin-bundle 3.x and won\'t be possible in 4.0.');
```

Comment:
```suggestion
        $this->expectDeprecation('Using "Sonata\AdminBundle\Admin\AbstractAdmin::getDataSourceIterator()" without setting a "Sonata\AdminBundle\Exporter\DataSourceInterface" instance in the admin is deprecated since sonata-admin/admin-bundle 3.x and won\'t be possible in 4.0.');
```